joomla/joomla-cms
# [4.0] Namespace com_checkin

Pull Request for Issue # .

### Summary of Changes
This simple PR convert com_checkin component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_checkin.zip )

2. Access to Content -> Articles, click on an article to edit, without saving (so that the article will be checked in)

3. Access to System -> Global Checkin, try to checkin the table, make sure it is working as expected.
